go/vt/sqlparser/parse_test.go

```diff
@@ -633,6 +636,9 @@ func TestValid(t *testing.T) {
 	}, {
 		input: "SELECT k FROM t1 GROUP BY k HAVING k = 'Müller' COLLATE latin1_german2_ci",
 		output: "select k from t1 group by k having k = 'Müller' collate latin1_german2_ci",
+	}, {
+		input: "SELECT k FROM t1 join t2 order by a COLLATE latin1_german2_ci, b COLLATE latin1_german2_ci",
+		output: "select k from t1 join t2 order by a collate latin1_german2_ci asc, b collate latin1_german2_ci asc",
```

Comment:
Prefer using lower-cased keywords. Then you don't have to specify an output. This also gives us the signal that the query will be unchanged.
Then, in cases where the output is different from the input, it will be obvious.